exporter/SynthesisFusionAddin/src/UI/ConfigCommand.py

```diff
@@ -361,7 +361,10 @@ def export(exporterOptions: moduleExporterOptions.ExporterOptions) -> None:
     exporterOptions.version = docVersion
     exporterOptions.materials = 0
 
-    Parser.Parser(exporterOptions).export()
+    try:
+        Parser.Parser(exporterOptions).export()
+    except:
+        return
```

Comment:
Yeah I think you should pass errors back to the web ui (using `html_args.returnData`) to avoid showing a success message